build.sbt

```
@@ -1,17 +1,31 @@
 lazy val scala212 = "2.12.18"
 lazy val scala213 = "2.13.7"
+lazy val scala213dot11 = "2.13.14"
```

nit: should this be dot14?
```suggestion
lazy val scala213dot14 = "2.13.14"
```